app/upgrades.go

```diff
@@ -361,3 +379,31 @@ func setExchangeParams(ctx sdk.Context, app *App) {
 	}
 	ctx.Logger().Info("Done ensuring exchange module params are set.")
 }
+
+// updateIbcMarkerDenomMetadata iterates markers and creates denom metadata for ibc markers
+// TODO: Remove with the saffron handlers.
+func updateIbcMarkerDenomMetadata(ctx sdk.Context, app *App) {
+	ctx.Logger().Info("Updating ibc marker denom metadata")
+	app.MarkerKeeper.IterateMarkers(ctx, func(record types.MarkerAccountI) bool {
+		if strings.HasSuffix(record.GetDenom(), "ibc/") {
```

Comment:
Just double checking that this is correct. Do ibc markers really **end** with `ibc/` ? That seems like it'd be a prefix, or that there'd be something like a `-` before the `ibc/`.